### PR TITLE
hyphens: auto for h3

### PR DESCRIPTION
**Pull request by itay1313**
for horizontal bug 
DX-105

### CODE REVIEW

File: css/inner-page.css [hunk 1]
```diff
@@ -71,6 +60,7 @@ h3[id] {
     font-size: 24px;
     line-height: 32px;
     margin-bottom: var(--space-unit);
+    hyphens: auto;
```
Comment (j9t) [inline css/inner-page.css:63]: We need something more sustainable, as this way, we run a risk of adding `hyphens: auto` to more and more elements.

For example, we could add it to `html` instead, then remove it from elements where we don’t want the behavior (like in the nav).

Please add a TODO that we review use of `hyphens`.

**Review by j9t — CHANGES_REQUESTED** [review]

File: css/inner-page.css (hunk 1 above)
Comment (itay1313) [reply css/inner-page.css:64]: done, will check about what you suggested

File: css/inner-page.css [hunk 2]
```diff
@@ -71,6 +60,8 @@ h3[id] {
     font-size: 24px;
     line-height: 32px;
     margin-bottom: var(--space-unit);
+    /* todo review use of hyphens */
```
Comment (j9t) [inline css/inner-page.css:63]: Thanks! Please use our standard syntax to make sure we can search for it case-sensitively.

**Review by j9t — CHANGES_REQUESTED** [review]

File: css/inner-page.css (hunk 2 above)
Comment (itay1313) [reply css/inner-page.css]: done

**Review by j9t — APPROVED** [review]

**Pull request merged**